charts/pwa/values.yaml

```diff
@@ -1,6 +1,13 @@
+# install VSCode plugin redhat.vscode-yaml
+# yaml-language-server: $schema=./values.schema.json
+
 # Default values for pwa-main.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
+
+updateStrategy: RollingUpdate
```

Comment:
The `README.md` of the pwa chart contains a description of all important characteristics that can be customized. I would argue that the newly created `updateStrategy` is such an important characteristic.